src/components/Sidebar.js

```diff
@@ -97,6 +97,9 @@ export default ({ showMobileSidebar, isJudgingOpen, isSubmissionsOpen, theme })
     links.push({ location: '/charcuterie', text: 'CHARCUTERIE' })
   }
 
+  // List the application as the last item on the menu
+  links.push({ location: '/application', text: 'APPLICATION' })
```

Comment:
This shouldn't always show up right? Only after nwhacks is released?

Reply:
Yea. I think we talked about feature flagging it - we'll need to add an enhancement to the cms, basically have an `application open` field to set when we want to release this tab.
I can work on the enhancement in the cms but can we address this in another PR?

Reply:
Coolcool